### DISCUSSION TITLE
Difference Between Mac Mouse Fix 1 and 2

### DISCUSSION THREAD

**Discussion by ghost** [noah-nuebling/mac-mouse-fix#133] [Questions] [answered]
Hi, could someone please explain how Mac Mouse 2 is different from the current public version? Thank you.

**Comment by noah-nuebling** [comment, marked as answer]
Hey @JuNkY14257, thanks for reaching out!

Here are the release notes for the first Mac Mouse Fix 2 Beta:

> - Click and drag to use Mission Control, switch between Spaces or see Application Windows from a single button in a highly responsive and intuitive way. This is made possible because due to reverse engineering the way that touch input is processed by macOS to enable for unparalleled simulation of 3 finger swipes on an Apple Trackpad.
> - Brand new, best-in-class remapping engine that lets you configure your mouse buttons in whatever way you want in a super easy way. Fully supports mice with up to 32 buttons. Use double click, click and hold, click and drag, keyboard modifiers and more to make your mouse super useful.
> - Added an option to use 'Smart Zoom' right from your mouse.

Since then, the only notable new feature I introduced remapping to keyboard shortcuts. 

You can look at the release notes for all version 2 betas on the [releases page](https://github.com/noah-nuebling/mac-mouse-fix/releases?after=2.0.0-Beta-4).
